ruff.toml

```diff
@@ -3,7 +3,7 @@ exclude = [
     ".venv",
     "__pycache__",
     "venv",
-    "providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/proto/*"
+    "providers/openfeature-provider-flagd/src/schemas/**"
```

Comment:
i think this is now not in sync with our specified path, and also needs an update